toolkits/global/packages/global-expander/HISTORY.md

```diff
@@ -1,5 +1,8 @@
 # History
 
+## 2.3.0 (2021-09-20)
```

Comment:
Should this be 4.1.0?

Reply:
@jon-stevens Whoops! Looks like it. Not sure what the agreed SN procedure is for a small fix on a merged PR?

Reply:
Ah I had the tab open since yesterday and didn't notice it had already been merged!
I suppose it can go in the next commit or a separate one?

Reply:
Ah I had the tab open since yesterday and didn't notice it had already been merged!
I suppose it can go in the next commit or a separate one?

Reply:
Argh! Sorry @Heydon 🤦 

My bad. I missed this yesterday. And this has just led me to also realise something else that is missing from your PR - you will need to update the version in each package's package.json file to match the version you have specified in the history.md.

Frontend Package Manager will only publish a package to npm if the following are present on the merge commit:
- there is a changelist of files/folders present for that package
- history.md has been updated
- package.json version property has been updated and is higher version that what is currently on npm

If you want to check to see if a package was published you can check the Travis logs for `info update package` which will show true or false for each package.

If you now separately commit the package.json changes for the packages you've changed you may find that the packages are not published to npm because frontend-package-manager doesn't detect any change to the history.md file in that new commit. There are two workarounds:
1. Make a trivial change to the history.md file in your package.json changes commit
2. Make the package.json change locally, publish manually from the CLI using `npm run publish`, then push your package.json changes to the repo whe complete

Option 1 is preferred because option 2 will mean the package is published to npm with you as the author instead of Springer Nature (which is no big deal but might as well not do that if poss).